samples/charts/financial-chart/annotations/Program.cs

```diff
@@ -25,8 +26,8 @@ public static async Task Main(string[] args)
             // registering Infragistics Blazor
             builder.Services.AddScoped(typeof(IInfragisticsBlazor), typeof(InfragisticsBlazor));
 
-            builder.Services.AddScoped(typeof(Samples.Services.StocksUtility));
-            builder.Services.AddScoped(typeof(Samples.Services.Stock));
+            builder.Services.AddScoped(typeof(Samples.Services.StockItem));
+            builder.Services.AddScoped(typeof(Samples.Services.StockList));
```

Comment:
as I explained before, do not add data models as Services because they are not service that fetch any data